app/helpers/Formatters.scala

```diff
@@ -39,8 +39,8 @@ trait DateFormatters {
 
 trait FileFormatters {
 
-  private val kbThreshold = 1000
-  private val mbThreshold = 1000000
+  private val kbThreshold = 1024
```

Comment:
Could you please write tests for these two threshold updates for the fileSize method?